app/kafka/manager/KafkaJMX.scala

```diff
@@ -0,0 +1,118 @@
+package kafka.manager
+
+import javax.management._
+import javax.management.remote.{JMXConnectorFactory, JMXServiceURL}
+
+object KafkaJMX {
+
+  def connect(jmxHost: String, jmxPort: Int): Either[ApiError, MBeanServerConnection] = {
```

Comment:
I have tried to keep the use of scalaz disjunction only in the KafkaManager.scala code.  Since this code is most likely going to be used in one of the actors, let's stick to using Try's if possible.  The ApiError object is mean for the KafkaManager which just provides an api/facade to the actors.